python/examples/advanced/Constraints_Suite.ipynb

```diff
@@ -82,143 +82,23 @@
-    "        # check for nans, Nones\n",
-    "        condition=lambda x: not x.max >= number,\n",
-    "        metric_selector=MetricsSelector(column_name=column_name, metric_name=\"distribution\"),\n",
-    "    )\n",
-    "    return constraint\n",
-    "\n",
-    "def stddev_between_range(column_name,lower,upper):\n",
-    "    \"\"\"Estimated standard deviation must be between range defined by lower and upper bounds.\n",
-    "\n",
-    "    Parameters\n",
-    "    ----------\n",
-    "    column_name : str\n",
-    "        Column the constraint is applied to\n",
-    "    lower : int\n",
-    "        Lower bound of defined range\n",
-    "    upper : int\n",
-    "        Upper bound of the value range\n",
-    "    \"\"\"\n",
-    "\n",
-    "    constraint = MetricConstraint(\n",
-    "        name=\"{} standard deviation between {} and {} (inclusive)\".format(column_name, lower, upper),\n",
-    "        condition=lambda x: not lower > x.stddev and not x.stddev > upper,\n",
-    "        metric_selector=MetricsSelector(column_name=column_name, metric_name=\"distribution\"),\n",
-    "    )\n",
-    "    return constraint\n",
-    "\n",
-    "def quantile_between_range(column_name:str, quantile: float, lower: float,upper: float, skip_missing: bool = True):\n",
-    "    \"\"\"Q-th quantile value must be withing the range defined by lower and upper boundaries.\n",
-    "\n",
-    "    Parameters\n",
-    "    ----------\n",
-    "    column_name : str\n",
-    "        Column the constraint is applied to\n",
-    "    quantile_value : float\n",
-    "        Quantile value. E.g. median is equal to quantile_value=0.5\n",
-    "    lower : float\n",
-    "        Lower bound of defined range\n",
-    "    upper : float\n",
-    "        Upper bound of the value range\n",
-    "    skip_missing : bool\n",
-    "        If skip_missing is True, missing distribution metrics will make the check pass.\n",
-    "        If False, the check will fail on missing metrics\n",
-    "    \"\"\"\n",
-    "\n",
-    "    def quantile_in_range(metric):\n",
-    "        if not metric.kll.value.is_empty():\n",
-    "            quantile_value = metric.kll.value.get_quantile(quantile)\n",
-    "            result = not lower > quantile_value and not quantile_value > upper\n",
-    "            return result\n",
-    "        else:\n",
-    "            return True if skip_missing else False\n",
-    "\n",
-    "    constraint = MetricConstraint(\n",
-    "    name=\"{} {}-th quantile value  between {} and {} (inclusive)\".format(column_name, quantile, lower, upper),\n",
-    "    condition = quantile_in_range,\n",
-    "    metric_selector=MetricsSelector(column_name=column_name, metric_name=\"distribution\"),\n",
-    "    )\n",
-    "    return constraint\n",
-    "\n"
+    "from whylogs.core.constraints import ConstraintsBuilder\n",
+    "from whylogs.core.constraints import (\n",
```

Comment:
shouldn't this be imported from .helpers, instead of directly from constraints?

Reply:
i have made it available through `whylogs.core.constraints.__init__.py`, trying to simplify its usage. do you suggest something else?

Reply:
In the other cells, it's being imported from `.helpers`, so at least we should standardize this.
I personally prefer being imported from `helpers`, to distinguish from the core Constraints classes, like the Builder, but that's up to you!